src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -434,6 +435,10 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
             $loader->load('form_debug.xml');
         }
 
+        if ($this->validatorConfigEnabled) {
```

Comment:
We also need to ensure that the `Symfony\Component\Validator\Validator\TraceableValidator` does exist to prevent issue when FrameworkBundle 3.4 is used with older versions of the Validator component.